README.md

```diff
@@ -74,4 +78,87 @@ creating a starter project for the Raspberry Pi Zero or Zero W. It will get you
 started with the basics like bringing up the virtual Ethernet interface,
 initializing the application partition, and enabling ssh-based firmware updates.
 
+## Linux kernel configuration notes
```

Comment:
I love that you documented this process. I know the plans for 4.9 are kind of in flux and it might just be leapfrogged. Is the plan to continue paring the kernel down in future versions too?

Reply:
I don't know of any other options unfortunately. It depends on what upstream does. Raspbian pairs down their kernel after building so I don't expect them to provide a more minimal starting point.

Reply:
I'd like to partner with you on this for the 4.14 version if you're interested. I did work on 4.9 when Raspbian first changed to it. I'd like to understand the steps to bump a Nerves system's kernel version and I can help to document the general process.